ci/cmd/maestro/kubernetes_tools.go

```diff
@@ -145,47 +145,60 @@ func SearchLogsForSuccess(kubeClient kubernetes.Interface, namespace string, pod
 		os.Exit(1)
 	}
 
-	// Poll for success
-	startedWaiting := time.Now()
-
 	go func() {
 		defer close(result)
 		defer logStream.Close() //nolint: errcheck,gosec
+
 		r := bufio.NewReader(logStream)
-		for {
-			line, err := r.ReadString('\n')
+		type readresult struct {
+			line string
+			err  error
+		}
+		readres := make(chan readresult)
```

Comment:
nit: `readres` to `readRes`

Reply:
Fixed. Only named it that way at first to avoid a name collision, but I guess I didn't have anything else named `readResult` anyway.

Reply:
Whoops, mixed this up with the other comment and missed this one. Both are fixed now.